hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/AlertsHandler.java

```diff
@@ -433,85 +410,47 @@ public Response deleteAlert(
     @PUT
     @Path("/delete")
     @Produces(APPLICATION_JSON)
-    @ApiOperation(value = "Delete alerts with optional filtering.",
-            notes = "Return number of alerts deleted. + \n" +
-                    "WARNING: If not criteria defined, it deletes all alerts stored in the system. + \n" +
-                    "Tags Query language (BNF): + \n" +
-                    "[source] \n" +
-                    "---- \n" +
-                    "<tag_query> ::= ( <expression> | \"(\" <object> \")\" " +
-                    "| <object> <logical_operator> <object> ) \n" +
-                    "<expression> ::= ( <tag_name> | <not> <tag_name> " +
-                    "| <tag_name> <boolean_operator> <tag_value> | " +
-                    "<tag_name> <array_operator> <array> ) \n" +
-                    "<not> ::= [ \"NOT\" | \"not\" ] \n" +
-                    "<logical_operator> ::= [ \"AND\" | \"OR\" | \"and\" | \"or\" ] \n" +
-                    "<boolean_operator> ::= [ \"==\" | \"!=\" ] \n" +
-                    "<array_operator> ::= [ \"IN\" | \"NOT IN\" | \"in\" | \"not in\" ] \n" +
-                    "<array> ::= ( \"[\" \"]\" | \"[\" ( \",\" <tag_value> )* ) \n" +
-                    "<tag_name> ::= <identifier> \n" +
-                    "<tag_value> ::= ( \"'\" <regexp> \"'\" | <simple_value> ) \n" +
-                    "; \n" +
-                    "; <identifier> and <simple_value> follow pattern [a-zA-Z_0-9][\\-a-zA-Z_0-9]* \n" +
-                    "; <regexp> follows any valid Java Regular Expression format \n" +
-                    "---- \n",
-            response = ApiDeleted.class)
+    @ApiOperation(value = "Delete alerts with optional filtering.", notes = "Return number of alerts deleted. + \n" +
```

Comment:
Same here, if not changes, let's maintain the same format, specially on large annotations.